src/bin/index.js

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env node
 
+if (!process.env.DEBUG) {
+  process.env.DEBUG = "phenomic:*"
+}
+
 // still in src/, handy to play
 if (__filename.indexOf("phenomic/src") > 1) {
```

Comment:
Yup. I noticed that. But I didn't change my local folder (stills statinamic`) and it throws the extract error on Appveyor when I build docs.